system/ui/kubernetes.yaml

```diff
@@ -14,7 +14,7 @@ spec:
       shareProcessNamespace: true
       containers:
       - name: starling-ui
-        image: uobflightlabstarling/starling-ui
+        image: uobflightlabstarling/starling-ui:latest
```

Comment:
Just leaving it blank should achieve ":latest" no?

Reply:
Thought it best to be consistent with the other configs and explicit!